bootstrap.sh

```diff
@@ -44,8 +44,10 @@ $top/dist/getversion.pl -M $top > $top/dist/vtag.map
 # The 'default' mode also builds some modules needed by C++ code.
 # The 'full' mode also builds modules needed by C++ tests.
 
-# must install parent pom first:
+# must install parent poms first:
 echo "Downloading all dependencies. This may take a few minutes with an empty Maven cache."
+cd container-dependency-versions && mvn_install
+cd ../parent && mvn_install && cd ..
```

Comment:
I would recommend expressing the two above lines like this:
```
(
  cd container-dependency-versions 
  mvn_install
)
(
  cd parent
  mvn_install
)
```
The code inside the parenthesis is executed in its own context and whatever you do inside wont affect the working directory of the rest of the script.